src/uber/UberTask.php

```diff
@@ -123,9 +123,23 @@ function ($v1, $v2) {
       }
 
       // prompt user to choose from menu
-      $fzf = id(new UberFZF())
-        ->requireFZF()
-        ->setMulti(50)
+      $fzf = id(new UberFZF());
+      if (!$fzf->isFZFAvailable()) {
+        $this->console->writeOut(
+          "<bg:red>** %s **</bg>\n<bg:red>** %s **</bg>\n"
+          "<bg:red>** %s **</bg> %s\n"
+          "<bg:red>** %s **</bg>\n<bg:red>** %s **</bg>\n",
+          pht('WARNING'),
```

Comment:
Remove those multiple empty WARNING lines. :)

Reply:
It is on purpose to make WARNING more visible...

Reply:
The warning message is inconsistent with whole Arcanist flow. Also it translates the message this particular warning is more important compared to other warnings.